aarone_backend/contacts/models.py

```diff
@@ -1,7 +1,15 @@
 from django.db import models
-
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+from .tasks import send_email_task, send_email_about_new_query_to_aaron_task
```

Comment:
I would remove the destination email holders name from the function name in `send_email_about_new_query_to_aaron_task`. I would also recommend that both emails be sent from the same function